mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -36,13 +36,16 @@ CLASS net/minecraft/class_57 net/minecraft/entity/Entity
 	FIELD field_1639 prevRenderZ D
 	FIELD field_1644 rand Ljava/util/Random;
 	FIELD field_1647 fire I
+	FIELD field_1652 skinUrl Ljava/lang/String;
+	FIELD field_1653 cloakUrl Ljava/lang/String;
```

Comment:
is this for capes :thoccc:

Reply:
Yes

Reply:
Cloak is the old name and the one used in the code, so I figured it would be like Tile